implementations/rust/daemon/src/config.rs

```diff
@@ -110,8 +123,9 @@ impl From<cli::Args> for Config {
         }
 
         cfg.role = match args.role() {
-            cli::ChannelRole::Initiator => Role::Initiator,
-            cli::ChannelRole::Responder => Role::Responder,
+            cli::ChannelRole::Source => Role::Initiator,
+            cli::ChannelRole::Sink => Role::Responder,
+            cli::ChannelRole::Router => Role::Router,
```

Comment:
Same consistency suggestion as before -- I don't think we should have different names between internal / external concerns (e.g. it's all external)